util/algorithm_identifier.go

```diff
@@ -84,3 +85,26 @@ func CheckAlgorithmIDParamNotNULL(algorithmIdentifier []byte, requiredAlgoID asn
 
 	return errors.New("RSA algorithm appears correct, but didn't match byte-wise comparison")
 }
+
+func GetPublicKeyOID(c *x509.Certificate) (asn1.ObjectIdentifier, error) {
```

Comment:
Can you add a Godoc comment describing the purpose and use of this function since it's exported?

Reply:
Godoc added.